contracts/controller.sol

```diff
@@ -57,7 +57,9 @@ contract Controller is IController, Ownable, Transferrable {
 
     /// @notice Constructor initializes the owner with the provided address.
     /// @param _ownerAddress_ address of the owner.
-    constructor(address payable _ownerAddress_) public Ownable(_ownerAddress_, false) {}
+    constructor(address payable _ownerAddress_) public {
```

Comment:
I guess we don't need to make the rest of it upgradable as we are using ens anyways.

Reply:
Yes, making the wallet upgradeable is the main goal, we don't really care about the rest.

Reply:
great!